app/models/order.rb

```diff
@@ -241,9 +241,14 @@ def add_to_stockit
 
   def self.search(search_text, to_designate_item)
     fetch_orders(to_designate_item)
-    .where(" code LIKE :query OR stockit_organisations.name LIKE :query OR
-      stockit_local_orders.client_name LIKE :query OR
-      stockit_contacts.first_name LIKE :query OR stockit_contacts.last_name LIKE :query OR
+    .where(" code ILIKE :query OR 
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.  …

code climate warning.